static/docs/user-guide/data-sharing/mounted-cache.md

```diff
@@ -0,0 +1,144 @@
+# Sharing Data Through a Mounted Cache
+
+We have seen already how to share data through a
+[mounted DVC storage](/doc/user-guide/data-sharing/mounted-storage). In that
+case we have a copy of the data on the DVC storage and at least one copy on each
+user project, since deduplication does not work across filesystems.
+
+However the data management can be further optimized if we use a shared cache.
+The idea is that instead of mounting the DVC storage from the server, we can
+directly mount the cache directory (`.dvc/cache/`). If all the users do this,
+then effectively they will be using the same cache directory (which is mounted
+from the NAS server). So, if one of them adds something to the cache, it will
+appear automatically to the cache of all the others. As a result, no `dvc push`
+and `dvc pull` are needed to share the data, just a `dvc checkout` will be
+sufficient.
+
+> ** ❗ Caution:** Deleting data from the cache will also make it disappear from
+> the cache of the other users. So, be careful with the command `dvc gc` (which
+> cleans obsolete data from the cache) and consult the other users of the
+> project before using this command.
+
+The optimization in data management comes from using the _symlink_ cache type.
+You can find more details about it in the page of
+[Large Dataset Optimization](https://dvc.org/doc/user-guide/large-dataset-optimization).
+
+## Mounted Cache Example
```

Comment:
I still think that explaining SSHFS here is way too much - too specific, takes too much time and distracts from the point even if expandable sections are being used

Reply:
> I still think that explaining SSHFS here is way too much - too specific, takes too much time and distracts from the point even if expandable sections are being used

This looks like your personal opinion. I don't think the same. I think that without some specific details and a concrete example, the explanation would too abstract and much more difficult to understand.

Reply:
> I think that without some specific details and a concrete example

I don't even disagree with this. It's a matter of number of details and also a matter of using 1% concrete example vs using 80% example. SSHFS is way too specific and we have too many details. It's like 50% of the page is about setting up this.